lib/Core/SpecialFunctionHandler.cpp

```diff
@@ -458,8 +458,11 @@ void SpecialFunctionHandler::handleIsSymbolic(ExecutionState &state,
                                 std::vector<ref<Expr> > &arguments) {
   assert(arguments.size()==1 && "invalid number of arguments to klee_is_symbolic");
 
+  ref<Expr> arg0 = executor.toUnique(state, arguments[0]);
+  
+
```

Comment:
You could remove at least one blank line and maybe extend `test/Feature/IsSymbolic.c`. Otherwise it looks good to me.